src/app/modules/item/services/item-datasource.service.ts

```diff
@@ -22,11 +22,13 @@ export class ItemDataSource implements OnDestroy {
   /* state to put outputted */
   private state = new BehaviorSubject<Ready<ItemData>|Fetching|FetchError>(fetchingState());
   state$ = this.state.asObservable();
-  item$ = this.state.pipe( // only fetched items, to be use in template as it cannot properly infer types
+  itemData$ = this.state.pipe( // only fetched items, to be use in template as it cannot properly infer types
     filter<Ready<ItemData>|Fetching|FetchError, Ready<ItemData>>(isReady),
-    map(s => s.data.item)
+    map(s => s.data)
   )
 
+  item$ = this.itemData$.pipe(map(s => s.item));
```

Comment:
No linebreak between state and itemData but one between itemdata and item?
Add semicolon to be consistent with the other lines.

Reply:
fixed